includes/components/class-admin.php

```diff
@@ -946,6 +949,312 @@ protected function get_extensions_tab() {
 		return $current_tab;
 	}
 
+	/**
+	 * Gets system report.
+	 *
+	 * @return array
+	 */
+	protected function get_report() {
+		global $wpdb;
+
+		// Get cURL version.
+		$curl_version = '-';
+
+		if ( function_exists( 'curl_version' ) ) {
+			$curl_version = hp\get_array_value( curl_version(), 'version' ) . ', ' . hp\get_array_value( curl_version(), 'ssl_version' );
+		}
+
+		// Get post types count.
+		$post_types = [];
+
+		foreach ( get_post_types() as $post_type_name => $post_type_label ) {
+			$post_count = array_sum( (array) wp_count_posts( $post_type_name ) );
+
+			if ( ! $post_count ) {
+				continue;
+			}
+
+			$post_types[ $post_type_name ] = [
+				'label' => $post_type_label,
+				'value' => $post_count,
+			];
+		}
+
+		ksort( $post_types );
+
+		// Get active plugins.
+		$active_plugins = array_filter(
+			array_map(
+				function ( $plugin ) {
+					$file = WP_PLUGIN_DIR . '/' . $plugin;
+
+					if ( file_exists( $file ) ) {
+						$headers = array_filter(
+							get_file_data(
+								$file,
+								[
+									'name'    => 'Plugin Name',
+									'version' => 'Version',
+									'url'     => 'Plugin URI',
+									'author'  => 'Author',
+								]
+							)
+						);
+
+						$value = [
+							'label' => hp\get_array_value( $headers, 'name' ),
+							'value' => sprintf( esc_html__( 'by %1$s - %2$s', 'hivepress' ), hp\get_array_value( $headers, 'author' ), hp\get_array_value( $headers, 'version' ) ),
+							'url'   => hp\get_array_value( $headers, 'url' ),
+						];
+
+						return $value;
+					}
+				},
+				(array) get_option( 'active_plugins' )
+			)
+		);
+
+		// Get theme info.
+		$theme = wp_get_theme( get_template() );
+
+		$theme_info = [
+			[
+				'label'    => esc_html__( 'Name' ),
+				'label_en' => 'Name',
+				'value'    => $theme->get( 'Name' ),
+			],
+			[
+				'label'    => esc_html__( 'Version' ),
+				'label_en' => 'Version',
+				'value'    => $theme->get( 'Version' ),
+			],
+			[
+				'label'    => esc_html__( 'Author URL', 'hivepress' ),
+				'label_en' => 'Author URL',
+				'value'    => $theme->get( 'AuthorURI' ),
+			],
+			[
+				'label'    => esc_html__( 'Child theme', 'hivepress' ),
+				'label_en' => 'Child theme',
+				'value'    => is_child_theme() ? '+' : '-',
+			],
+		];
+
+		if ( is_child_theme() ) {
+			$parent_theme = wp_get_theme( $theme->template );
+
+			$theme_info = array_merge(
+				$theme_info,
+				[
+					[
+						'label'    => esc_html__( 'Parent Theme Name', 'hivepress' ),
+						'label_en' => 'Parent Theme Name',
+						'value'    => $parent_theme->name,
+					],
+					[
+						'label'    => esc_html__( 'Parent Theme Version', 'hivepress' ),
+						'label_en' => 'Parent Theme Version',
+						'value'    => $parent_theme->version,
+					],
+				]
+			);
+		}
+
+		$theme_info[] = [
+			'label'    => esc_html__( 'HivePress support', 'hivepress' ),
```

Comment:
As noted before, please don't add brand names to translatable strings, since it makes no sense to translate them anyway and "Google Maps Support" with "HivePress Support" would require double translation, use placeholders instead: %s support